src/@next/Radio/Radio.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+import { Label, LabelWrapper, RadioInput, RadioWrapper } from './RadioStyle';
+import { Typography } from '../Typography';
+import { v4 as uuidv4 } from 'uuid';
+import { Colors } from '..';
+
+export type RadioProps = React.HTMLAttributes<HTMLInputElement> & {
+  label?: string;
+  helperText?: string;
+  disabled?: boolean;
+  name?: string;
+  checked?: boolean;
+};
+
+export const Radio = ({ ...props }: RadioProps) => {
```

Comment:
can we destructure the props that were used below. it'll be easier to read and we will have lesser steps to access the prop. we are also doing similarly on the other components

Reply:
Updated